client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -831,6 +807,56 @@ text.slicetext {
   text-shadow: 1px 1px 5px #333;
 }
 
+.pagination {
```

Comment:
This is styling AntPagination to look like our old paginator? If so, this should be in `ant.less`, no?

Reply:
There were two CSS overrides in this file, I just merged them together and added some new rules for Ant. I'm not sure that we should move it to `ant.less`

Reply:
But are these overrides apply to the Ant paginator too?

Reply:
Yes

Reply:
If it matters - we don't use "raw" bootstrap paginator anywhere - just our Paginator component

Reply:
In this case it's worth removing unneeded styles and moving this to the Ant file. 

cc: @ranbena

Reply:
I think our styles need more solid refactoring and it's not a subject of this PR